pkg/render/applicationlayer/envoy-config.yaml.template

```diff
@@ -1,3 +1,40 @@
+{{- define "access_log" -}}
```

Comment:
Another one for the backlog - we want do be doing conditional logic in golang rather than templates. This one seems borderline, but if this gets much more complex we'll definitely need to put this in golang so we can more easily unit test this.

Reply:
Noted - will create a ticket for further refactoring.